docs/getting-started.md

```diff
@@ -1 +1,137 @@
-This is a placeholder for some docs
+## 1. Overview
+Kubeflow is a novel open source tool for Machine Learning workflow orchestration on Kubernetes.
+It has great powers, but deploying it may not be so easy, depending on how and where you deploy
+your Kubernetes. This tutorial will show you an easy way to deploy Kubeflow using MicroK8s, a
+lightweight version of Kubernetes, in a few simple steps.
+
+### What you'll learn
+
+* How to deploy MicroK8s on Ubuntu
+* How to deploy Kubeflow using MicroK8s
+* How to access your Kubeflow dashboard
+
+### What you'll need
+* Desktop or Virtual Machine with Ubuntu 16.04 or above
+* A minimum of 4 CPU, 14GB RAM, 50GB Disk (recommended 8 CPU, 16GB RAM, 60GB Disk)
+* Hyper V, if using Multipass on a Windows machine (not available on Windows 10 Home)
+* Some basic command-line knowledge
+
+
+## 2. Open Ubuntu terminal
+
+All we need to do here is access the terminal of the machine we want to deploy Kubeflow to. This
+is fairly straightforward.
+
+### Option 1: On Ubuntu machine
+
+1. Open terminal (CTRL + ALT + t)
+
+### Option 2: Accessing remote Ubuntu virtual machine
+
+1. Spin up a virtual machine with Ubuntu (on a server or cloud)
+
+1. Access the machine terminal using `ssh`:
+
+       $ ssh ubuntu@<machine-ip-address>
+
+### Option 3: On Windows or MacOS
```

Comment:
Multipass isn't the recommended install any more. I would suggest just covering the easiest use case (snap for Linux) and redirecting anyone else to the MicroK8s docs, as otherwise you will also need to track all the changes

Reply:
Oh, sorry, I get it, this is to get Ubuntu. Is this really necessary?

Reply:
That's a good question. @tvansteenburgh, are the windows and macos version of microk8s stable enough that we can just recommend those?

Reply:
> That's a good question. @tvansteenburgh, are the windows and macos version of microk8s stable enough that we can just recommend those?

Yes.